internal/opa/output.go

```diff
@@ -83,13 +85,22 @@ func prepTemplateFields(a *ast.AnnotationsRef) TemplateFields {
 		// conventions i.e. "package policy.<type>.<name>"
 		shortPath = trimmedPath
 	}
+
+	shortName := getShortName(a)
+
+	var docsUrl string
+	if len(pathStrings) > 4 && shortName != "" && (strings.HasPrefix(trimmedPath, "policy.pipeline.") || strings.HasPrefix(trimmedPath, "policy.release.")) {
```

Comment:
Why greater than 4?

Reply:
Hmm yeah, probably should be `len(..) == 4`

Reply:
Actually 5, because it still include "data" as element 0.

Reply:
Ah so this is checking if it includes a certain prefix?. If so, maybe something like this is more easily read: `len(pathStrings) > len("myprefix")`. Or even better `strings.HasPrefix(pathStrings, "myprefixX")`, if `X` is always the same value.